.github/workflows/ci-jobs.yml

```diff
@@ -138,17 +114,22 @@ jobs:
       with:
         repository: ${{ inputs.checkout-repo }}
         ref: ${{ inputs.checkout-ref }}
-        submodules: true
-        fetch-depth: 0
```

Comment:
I think we still need these for the .gitmodules configuration to work on release branches.